src/apps/common/submodules/chooseModel/chooseModel.js

```diff
@@ -39,19 +39,26 @@ define(function(require) {
 					brands: []
 				},
 				families,
-				models;
-
+				models,
+				prefix = ['vvx'];
 			_.each(data, function(brand, brandKey) {
 				families = [];
 
 				_.each(brand.families, function(family, familyKey) {
 					models = [];
 
 					_.each(family.models, function(model, modelKey) {
-						models.push({
-							id: modelKey.toLowerCase(),
-							name: model.name
-						});
+						if ( prefix.includes(family.name) ) {
```

Comment:
Since there is no a big diference between the if/else statement I think this can be a little short, like this
```
models.push({
    id: modelKey.toLowerCase(),
    name: prefix.includes(family.name) ? _.toUpper(family.name) + " " + model.name : model.name
});
```

by this way, the if/else statement are not needed

Reply:
changed